services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -232,24 +272,42 @@ class ApplicationServiceITest : DatabaseTest() {
         verify { cableReportServiceAllu wasNot Called }
     }
 
+    @Test
+    fun `getAllApplicationsForUser returns applications for user hankkeet`() {
+        assertThat(applicationRepository.findAll()).isEmpty()
+        val hanke = hankeRepository.save(HankeEntity(hankeTunnus = "HAI-1234"))
+        val hanke2 = hankeRepository.save(HankeEntity(hankeTunnus = "HAI-1235"))
+        val hanke3 = hankeRepository.save(HankeEntity(hankeTunnus = "HAI-1236"))
+
+        permissionService.setPermission(hanke2.id!!, username, Role.KATSELUOIKEUS)
+
+        val application1 = alluDataFactory.saveApplicationEntity(username = username, hanke = hanke)
+        val application2 =
+            alluDataFactory.saveApplicationEntity(username = "secondUser", hanke = hanke2)
+        alluDataFactory.saveApplicationEntity(username = "thirdUser", hanke = hanke3)
+
+        assertThat(applicationRepository.findAll()).hasSize(3)
+        val response = applicationService.getAllApplicationsForUser(username).map { it.id }
+        assertThat(response).containsExactlyInAnyOrder(application1.id, application2.id)
```

Comment:
While I'm not a puritan about AAA-pattern (Arrange, Act, Assert), I find it very helpful to have the Act part as a separate code block, i.e. surrounded by empty lines. Makes it easier to spot what's actually being tested.
```suggestion
        assertThat(applicationRepository.findAll()).hasSize(3)
        
        val response = applicationService.getAllApplicationsForUser(username)
                
        assertThat(response.map { it.id }).containsExactlyInAnyOrder(application1.id, application2.id)
```

Reply:
I don't see the fix.